pkg/controller/instance/instance_controller.go

```diff
@@ -291,31 +286,21 @@ func (r *Reconciler) getOperatorVersion(instance *kudov1alpha1.Instance) (ov *ku
 	return ov, nil
 }
 
-func getParameters(instance *kudov1alpha1.Instance, operatorVersion *kudov1alpha1.OperatorVersion) (map[string]string, error) {
+func getParameters(instance *kudov1alpha1.Instance, operatorVersion *kudov1alpha1.OperatorVersion) map[string]string {
 	params := make(map[string]string)
 
 	for k, v := range instance.Spec.Parameters {
 		params[k] = v
 	}
 
-	missingRequiredParameters := make([]string, 0)
-	// Merge defaults with customizations
+	// Merge defaults with customizations, if no override exist, use the default parameter
```

Comment:
maybe say "instance parameters" instead of customizations? I find that better to understand